package.json

```diff
@@ -66,6 +67,7 @@
     "@angular/cli": "^7.3.1",
     "@angular/compiler-cli": "^7.2.4",
     "@nativescript/schematics": "^0.5.0",
+    "@netbasal/spectator": "^3.6.1",
```

Comment:
This is something we want/need?

Reply:
Yes, it makes testing component lifecycle hooks (requirements of host-components) much easier.

Reply:
Ok